.travis.yml

```diff
@@ -80,6 +80,31 @@ matrix:
         - go get github.com/bazelbuild/buildtools/buildifier
         - ./ci/travis/bazel-format.sh
 
+    - os: linux
+      env: SANITIZER=1 CC=clang PYTHON=3.5 PYTHONWARNINGS=ignore
+
+      install:
+        - eval `python $TRAVIS_BUILD_DIR/ci/travis/determine_tests_to_run.py`
+        - if [ $RAY_CI_PYTHON_AFFECTED != "1" ]; then exit; fi
+
+        - ./ci/suppress_output ./ci/travis/install-bazel.sh
+        - ./ci/suppress_output ./ci/travis/install-dependencies.sh
+        - export PATH="$HOME/miniconda/bin:$PATH"
+        - ./ci/suppress_output ./ci/travis/install-ray.sh
+
+        # Run raylet integration tests with address sanitizer
+        - export RAY_BAZEL_CONFIG=--config=asan
+        - ./ci/suppress_output bash src/ray/test/run_core_worker_tests.sh
+        - ./ci/suppress_output bash src/ray/test/run_object_manager_tests.sh
+
+        # Run raylet integration tests with thread sanitizer
+        - export RAY_BAZEL_CONFIG=--config=tsan
```

Comment:
This is going to leak one of the `RAY_BAZEL_CONFIG` across the subsequent statements, right? If it needs to be an environment variable, it would be better to scope it to them if possible. Maybe like:

	(
		export RAY_BAZEL_CONFIG=...
		...
		...
	)

But maybe that's not necessary? Can we just pass these as arguments to be forwarded to Bazel inside each script (via `"$@"`)? It'd be nice to keep the arrays intact (the space splitting of `$RAY_BAZEL_CONFIG` makes me uncomfortable).

Reply:
Thanks, I fixed it a bit differently in https://github.com/ray-project/ray/pull/6701 which replaces this PR (instead of making it a global variable, I make it a local one). Not sure what you mean with space splitting.

Reply:
Oh I was just referring to the fact that missing quotes cause splitting on spaces, which is generally bad.